website/blog/2017-11-07-1.8.0.md

```diff
@@ -0,0 +1,668 @@
+```
+
+Output:
+
+```
+Voilà! In view, a humble vaudevillian veteran cast vicariously as both victim
+and villain by the vicissitudes of Fate. This visage, no mere veneer of vanity,
+is a vestige of the vox populi, now vacant, vanished. However, this valourous
+visitation of a bygone vexation stands vivified and has vowed to vanquish these
+venal and virulent vermin vanguarding vice and vouchsafing the violently vicious
+and voracious violation of volition! The only verdict is vengeance; a vendetta
+held as a votive, not in vain, for the value and veracity of such shall one day
+vindicate the vigilant and the virtuous. Verily, this vichyssoise of verbiage
+veers most verbose, so let me simply add that it's my very good honour to meet
+you and you may call me V.
+```
+
+> ~~Note: We're considering adding an option for this, check [#3183](https://github.com/prettier/prettier/pull/3183) for discussion.~~
+> Update: We've added an option in `1.8.2` called [`--no-prose-wrap`](https://prettier.io/docs/en/options.html#prose-wrap)
+
+> Note for CJK users: If your markdown renderer does not support CJK line ending, you'll have to use plugin like [markdown-it-perfect-newline-for-cjk](https://www.npmjs.com/package/markdown-it-perfect-newline-for-cjk), [hexo-filter-fix-cjk-spacing](https://www.npmjs.com/package/hexo-filter-fix-cjk-spacing), etc. to remove additional spaces.
+>
+> ```
+> // Source
+> 一二三
+> 四五六
+> 七八九
+>
+> // Rendered content with unsupported renderer
+> 一二三 四五六 七八九
+>
+> // Rendered content with supported renderer or via plugin
+> 一二三四五六七八九
+> ```
+
+**Code Formatting**
+
+Powered by Prettier's generic "multiparser", Prettier will format code blocks in Markdown! We use the language code provided with the code block to determine which language it is, and thus we can format any language that Prettier supports (including Markdown itself, if you're into that).
+
+Input:
+
+<!-- prettier-ignore -->
+````md
+```js
+reallyUgly    (
+javascript
+  )
+```
+
+```css
+.h1 {     color : red }
+```
+````
+
+Output:
+
+````md
+```js
+reallyUgly(javascript);
+```
+
+```css
+.h1 {
+  color: red;
+}
+```
+````
+
+<!-- prettier-ignore -->
+> Note: In some cases you may not want to format your code in Markdown, and just like in other languages, in Markdown you can use an ignore comment before the code block to ignore it from formatting:
+>
```

Comment:
I guess the above `<!-- prettier-ignore -->` should be here?

Reply:
Bug, raised #3609